Resources/Prototypes/Traits/inconveniences.yml

```
@@ -83,3 +83,18 @@
+  category: Physical
+  points: 6
+  requirements:
+  functions:
+    - !type:TraitAddComponent
+      component:
+    - type: DamageOverTime
+      damage:
+        types:
+          Piercing: 0.3
+      interval: 1.1
+      ignoreResistances: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)